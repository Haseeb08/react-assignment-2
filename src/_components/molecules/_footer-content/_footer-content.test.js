import React from "react";
import { create } from "react-test-renderer";
import {
  render,
  screen,
  getByRole,
  getByPlaceholderText,
  fireEvent,
} from "@testing-library/react";
import "@testing-library/jest-dom/extend-expect";
import FooterContent from "./_footer-content";

describe("Add Image box testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<FooterContent />);
    expect(input.toJSON()).toMatchSnapshot();
  });

});
