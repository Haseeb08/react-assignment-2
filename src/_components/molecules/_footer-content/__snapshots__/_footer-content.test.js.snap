// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Image box testing Matches the snapshot 1`] = `
<div
  className="makeStyles-content-1"
  data-testid="footer"
>
  <div>
    <p
      className="MuiTypography-root makeStyles-text-2 MuiTypography-body1"
    >
      2020 Copyright 
        :  
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-3 MuiTypography-colorPrimary"
        href="https://github.com/Haseeb08/"
        onBlur={[Function]}
        onFocus={[Function]}
      >
        git-haseeb
      </a>
    </p>
  </div>
</div>
`;
